network-api/networkapi/wagtailpages/models.py

```diff
@@ -548,9 +549,22 @@ def get_entries(self, context=dict()):
         """
         entries = self.get_all_entries()
         if hasattr(self, 'filtered'):
-            entries = self.filter_entries_for_tag(entries, context)
+            entries = self.filter_entries(entries, context)
         return entries
 
+    def filter_entries(self, entries, context):
+        type = self.filtered.get('type')
```

Comment:
...look up 5 lines, where you will find the `hasattr`? =D

Good point about the reserved word, though. Then again, flake8 doesn't warn about it, so I'm also tempted to say that if this was a PEP policy, it would have flagged that?

Reply:
if `filter_entries` is only being called within this class and we know that the caller is always going to do the `hasattr` check, yep, this should be fine!